script.js

```diff
@@ -6,4 +6,62 @@ document.querySelector(".trigger").addEventListener("click", e => {
 
 document.querySelector(".close-button").addEventListener("click", e => {
     modal.classList.toggle("show-modal");
-});
\ No newline at end of file
+});
+
+function modalEndGame () {
```

Comment:
not a great function name. the code toggles the endgame modal, so maybe `toggleEndGameModal`? also, no space after function name before `()`